src/Mod/Fem/femobjects/constraint_initialpressure.py

```diff
@@ -0,0 +1,56 @@
+# ***************************************************************************
+# *   Copyright (c) 2022 Uwe Stöhr <uwestoehr@lyx.org>                      *
+# *                                                                         *
+# *   This file is part of the FreeCAD CAx development system.              *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "FreeCAD FEM constraint initial pressure document object"
+__author__ = "Uwe Stöhr"
+__url__ = "https://www.freecadweb.org"
```

Comment:
__url__ = "https://www.freecad.org/"

Reply:
Hmm, I am reluctant. We can do this change but then _all_ files should be updated. At the moment they are all use freecadweb.org and my PR should only add the constraint, not introduce other changes.

If you like, can you make a PR where you make the change for all files? Please bring this to the developer's list if the others agree.

Reply:
No thanks, I'm waiting for after toponaming merge to make big sweeping changes.